src/Controllers/MoviesController.cs

```diff
@@ -19,7 +21,38 @@ public IUnitOfWork UnitOfWork
         }
         public IActionResult Index()
         {
-            return View ();
+
+            var movies = _unitOfWork.MovieRepository.GetAll();
+
+            return View(movies);
```

Comment:
if we are using view models, you have to send MovieViewModel to the view instead Movie.